<?xml version="1.0"?>
<doc>
    <assembly>
        <name>Hangfire.Pro</name>
    </assembly>
    <members>
        <member name="T:Hangfire.CreateBatchFailedException">
            <summary>
            The exception that is thrown when a <see cref="T:Hangfire.BatchJobClient"/> class instance
            can not create a batch due to an exception.
            </summary>
        </member>
        <member name="M:Hangfire.CreateBatchFailedException.#ctor(System.String,System.Exception)">
            <summary>
            Initializes a new instance of the <see cref="T:Hangfire.CreateBatchFailedException"/>
            class with a specified error message and a reference to the
            inner exception that is the cause of this exception.
            </summary>
            <param name="message">The error message that explains the reason for the exception.</param>
            <param name="inner">The exception that is the cause of this exception, not null.</param>
        </member>
    </members>
</doc>
